e2e/runner/bitcoin.go

```diff
@@ -198,8 +198,11 @@ func (r *E2ERunner) SendToTSSFromDeployerWithMemo(
 		scriptPubkeys[i] = utxo.ScriptPubKey
 	}
 
+	// use static fee 0.0005 BTC to calculate change
 	feeSats := btcutil.Amount(0.0005 * btcutil.SatoshiPerBitcoin)
-	amountSats := btcutil.Amount(amount * btcutil.SatoshiPerBitcoin)
+	amountInt, err := zetabitcoin.GetSatoshis(amount)
```

Comment:
existing calculation `btcutil.Amount(amount * btcutil.SatoshiPerBitcoin)` has precision issue, so we use the conversion function `GetSatoshis`.